consent-banner.es5.js

```diff
@@ -184,5 +228,37 @@ window.BarnardosConsent = function(options) {
       sendClickAction(e.target);
     });
   }
-};
 
+  if (cookieOverlay) {
+    cookieOverlay.addEventListener("click", e => {
```

Comment:
Can't use arrow functions in ES5. This (and the next few instances) need to be changed to
```js
cookieOverlay.addEventListener("click", function(e) (
```